.travis.yml

```diff
@@ -12,7 +12,3 @@ android:
 
 licenses:
   - '.+'
-
-before_script:
```

Comment:
This won't be needed if I wrote the Gradle stuff below correctly (if afterEvaluate is the right spot)